src/game_mechanics/constants.tsx

```diff
@@ -0,0 +1,14 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+const variableDefaults = {
+  honey: 0,
+  bees: 0,
+  costOfNextBee: 1,
+  nectar: 0
+};
+
+const NECTAR_BY_BEE = 3;
+
+export { variableDefaults, NECTAR_BY_BEE };
```

Comment:
Like you note in your PR, we should definitely discuss how we want to modularize so we can start moving this kind of stuff outside the `App.tsx` file. I like the idea of making these constants in a file like this - that said, I'm not sure about the name `game_mechanics` for the folder. Especially when we start adding more constants like this, having it all in one file (and one big `export`) might get unwieldy. Since this is the only file like this yet, maybe we could maintain a `constants` folder and have different files that correspond to different `components`? (see below)

Reply:
I'm not good at naming things. I prefer your suggestions and will rename as such!